modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -86,7 +94,90 @@ public void create(
 			_objectScopeProviderRegistry.getObjectScopeProvider(
 				_objectDefinition.getScope());
 
-		if (objectScopeProvider.isGroupAware()) {
+		String importCreatorStrategy = (String)parameters.getOrDefault(
+			"importCreatorStrategy", "OVERWRITE_CREATOR");
+
+		if (importCreatorStrategy.equals("KEEP_CREATOR") &&
```

Comment:
Maybe we could implement `ImportTaskPreAction` and `ImportTaskPostAction` for this purpose. Registering those elements within the Object REST API modules would help us by delegating the set of the threadlocal to those implementations. So those classes would need to check if the item is an ObjectEntry and also if the parameter is true and also if the FF is active. And then, set and unset the thread local